File: src/main/java/nu/peg/slack/pt/api/slack/model/ErrorAttachment.java
package nu.peg.slack.pt.api.slack.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ErrorAttachment implements Attachment {

    private String color, text;

    @SerializedName("mrkdwn_in")
    private List<String> markdownIn;

}

File: src/main/kotlin/nu/peg/slack/pt/api/slack/model/ErrorAttachment.kt
package nu.peg.slack.pt.api.slack.model

import com.google.gson.annotations.SerializedName

import lombok.*

@Data
@NoArgsConstructor
@AllArgsConstructor
class ErrorAttachment : Attachment {

    private val color: String? = null
    private val text: String? = null

    @SerializedName("mrkdwn_in")
    private val markdownIn: List<String>? = null

}
